tensorflow/core/kernels/fused_batch_norm_op.cc

```diff
@@ -1963,6 +1963,7 @@ REGISTER_KERNEL_BUILDER(Name("FusedBatchNormGradV3")
                             .TypeConstraint<float>("U"),
                         FusedBatchNormGradOpV3<CPUDevice, Eigen::half, float>);
 
+
```

Comment:
Is this newline added by clang-format? If it didn't break any formatting checks, I don't think we need a newline here.

Reply:
sorry, i added by mistake on this PR instead of another ... reverting now.

Reply:
i have fixed it now.